### DISCUSSION TITLE
Add send later button to the send all farms

### DISCUSSION THREAD

**Discussion by TravianPlayer** [adipiciu/Travian-scripts#60] [General]
Hey

Travian has added a button to send all farmlists at once.
Can you make a hotfix and add send later button to it 
![لقطة شاشة 2024-01-17 191921](https://github.com/adipiciu/Travian-scripts/assets/156936235/66f2e51a-9630-462b-ad97-a72b88932cff)

**Comment by adipiciu** [comment]
Added in TTQ 2.0.12
